wordpress-mobile/WordPress-Android
# Implement "Follow other sites" QS task

Fixes #13846

This task implements the "Follow other sites" Quick Start task, which is part of the "Grow your audience" category.

This task has two steps: the first one shows a snackbar message and focus point on top of the "Reader" button on the bottom navigation bar and the second one shows a snackbar message on the Reader screen. The task is completed as soon as the Reader screen is shown.

https://user-images.githubusercontent.com/830056/106625548-84d35500-6555-11eb-980f-be8503056b10.mp4

## To test

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen and select a site where Quick Start is active.
1. Under the "Grow your audience" category, tap the "Follow other sites" task.
1. Notice that:
   1. A focus point appears over the Reader button on the bottom navigation bar.
   1. The correct snackbar message appears.
1. Tap on the "Follow other sites" task a second time.
1. Notice that there's still just one focus point over the Reader button.
1. Tap another task, like "Set your site title".
1. Notice that the focus point on top of the Reader button disappears.
1. Tap on the "Follow other sites" task again.
1. Tap on the Reader button.
1. On the Reader screen, notice that the correct snackbar message appears.
1. Go back to the My Site screen and scroll to the "Grow your audience" category.
1. Notice that the "Follow other sites" task is now completed.

PR submission checklist:
